src/script.c

```diff
@@ -359,6 +371,17 @@ static int scriptVerifyWriteCommandAllow(scriptRunCtx *run_ctx, char **err) {
         return C_ERR;
     }
 
+    /* Don't accept write commands if there are not enough good slaves and
+     * user configured the min-slaves-to-write option. */
```

Comment:
Maybe we should comment here that this is only for eval scripts that does not declares flags?